### DISCUSSION TITLE
MIT No Attribution

### DISCUSSION THREAD

**Discussion by Artem13327** [community/community#35221] [New to GitHub]
I propose to add to the selection of licenses MIT No Attribution

**Comment by airtower-luna** [comment]
This might be what you're looking for, basically a public domain dedication: https://choosealicense.com/licenses/unlicense/

**Reply by stokito** [reply]
This is a very bad suggestion, see https://github.com/orgs/community/discussions/53778

**Comment by stokito** [comment]
The license is not well known and should be avoided to not make a fragmentation. it wasn't really studied and questioned in courts
